cmd/notary/main_test.go

```diff
@@ -103,6 +103,7 @@ var exampleValidCommands = []string{
 	"status repo",
 	"publish repo",
 	"add repo v1 somefile",
+	"addhash repo targetv1 --sha256 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 10",
```

Comment:
this works too but so that we don't have to use flags, what I was suggesting was something like distribution uses: `sha256:aaaaaa...` then we'd just split on `:`. Means a user can add any hash, even one we don't necessarily recognize which they can validate in their own client integration.